ami_bag/ami_bag_constants.py

```diff
@@ -63,7 +63,7 @@
     "film": (set([PM_DIR, MZ_DIR, SC_DIR, IM_DIR]),
         set([JSON_EXT, MKV_EXT, MOV_EXT, MP4_EXT, JPEG_EXT, JPG_EXT, GZ_EXT, SRT_EXT, SCC_EXT])),
     "video": (set([PM_DIR, SC_DIR, IM_DIR]),
-        set([JSON_EXT, MOV_EXT, MKV_EXT, DV_EXT, MP4_EXT, JPEG_EXT, JPG_EXT, GZ_EXT, SRT_EXT, SCC_EXT])),
+        set([JSON_EXT, MOV_EXT, MKV_EXT, DV_EXT, MP4_EXT, JPEG_EXT, JPG_EXT, GZ_EXT, SRT_EXT, SCC_EXT, ISO_EXT])),
```

Comment:
This would look nicer if ISO had been added earlier in the list, and also hide that I forgot to add it earlier in development.